samples/snippets/src/main/java/dialogflow/cx/ListPages.java

```diff
@@ -39,23 +39,27 @@ public static void main(String[] args) throws IOException {
   // Lists all pages from the provided parameters
   public static void listPages(String projectId, String agentId, String flowId, String location)
       throws IOException {
-    PagesClient client = PagesClient.create();
-    Builder listRequestBuilder = ListPagesRequest.newBuilder();
-
-    listRequestBuilder.setParent(
-        "projects/"
-            + projectId
-            + "/locations/"
-            + location
-            + "/agents/"
-            + agentId
-            + "/flows/"
-            + flowId);
-    listRequestBuilder.setLanguageCode("en");
-
-    // Make API request to list all pages in the project
-    for (Page element : client.listPages(listRequestBuilder.build()).iterateAll()) {
-      System.out.println(element);
+    // Note: close() needs to be called on the PagesClient object to clean up resources
+    // such as threads. In the example below, try-with-resources is used,
+    // which automatically calls close().
+    try (PagesClient client = PagesClient.create()) {
+      Builder listRequestBuilder = ListPagesRequest.newBuilder();
+
+      listRequestBuilder.setParent(
+          "projects/"
+              + projectId
+              + "/locations/"
+              + location
+              + "/agents/"
+              + agentId
+              + "/flows/"
+              + flowId);
```

Comment:
Ok for now, but in the future, consider using `String.format()` as it's more conducive to understanding what's going on.